src/NuGet.Clients/NuGet.SolutionRestoreManager/IVsNuGetProgressReporter.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using NuGet.Commands;
+
+namespace NuGet.SolutionRestoreManager
+{
+    public interface IVsNuGetProgressReporter : IRestoreProgressReporter
```

Comment:
Any reason to not have xml doc comments on this interface?  I see you added them to other ones

Reply:
Honestly, no great reason :D 

I think it makes sense to add some docs here as well.